spec/models/spree/billing_integration_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe 'Deprecated BillingIntegration' do
+  before { Gem::Deprecate.skip, @original = true, Gem::Deprecate.skip }
+  after { Gem::Deprecate.skip = @original }
+
+  it 'returns the an instance scoped to PaymentMethod instead' do
+    payment_methods.each do |payment_method|
+      subject = build_class('BillingIntegration', payment_method).new
+      expect(subject).to be_a_kind_of(build_class('PaymentMethod', payment_method))
+    end
+  end
+
+  it 'passes the attributes on to the new instantiation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.